backend/src/data/models/chat-message/chat-message.model.ts

```diff
@@ -0,0 +1,41 @@
+import { Model, RelationMappings } from 'objection';
+
+import { DbTableName } from '~/common/enums/enums';
+
+import { Abstract } from '../abstract/abstract.model';
+import { User } from '../models';
+
+class ChatMessage extends Abstract {
+  public 'message': string;
+
+  public 'senderId': number;
+
+  public 'receiverId': number;
+
+  public static override get relationMappings(): RelationMappings {
+    return {
+      sender: {
+        relation: Model.HasOneRelation,
```

Comment:
For me it is hard to get the idea of the relation here. Do you want to change it because the User can have a message?
I chose HasOneRelation because I think that the message has a sender and receiver

Reply:
https://vincit.github.io/objection.js/guide/relations.html
Maybe this will help you, it helped me a lot

Reply:
I have read that article before but did not get the idea properly. Now, it seems, that I got it. Thanks!